src/metatensor/models/experimental/soap_bpnn/train.py

```diff
@@ -48,20 +48,29 @@ def train(
     continue_from: Optional[str] = None,
     output_dir: str = ".",
 ):
+    # TODO: check that we're only training on energy targets.
+    # also in alchemical
+
     all_species = get_all_species(train_datasets + validation_datasets)
     outputs = {
         key: ModelOutput(
             quantity=value.quantity,
             unit=value.unit,
-            per_atom=False,
+            per_atom=True,
         )
         for key, value in dataset_info.targets.items()
     }
+    # the model is always capable of outputting the last layer features
+    outputs["mts-models::aux::last_layer_features"] = ModelOutput(
+        quantity="",
+        unit="",
+        per_atom=True,
+    )
```

Comment:
If you leave some fields to the default, you don't need to specify them

```suggestion
    outputs["mts-models::aux::last_layer_features"] = ModelOutput(per_atom=True)
```